README.md

```diff
@@ -5,10 +5,10 @@ This repository contains Dockerfiles for MariaDB images for OpenShift and genera
 Users can choose between RHEL, Fedora and CentOS based images.
 
 MariaDB container is very similar to the MySQL container available at
-[https://github.com/sclorg/mysql-container](https://github.com/sclorg/mysql-container).
+[https://github.com/sclorg/mariadb-container](https://github.com/sclorg/mariadb-container).
```

Comment:
I think this was replaced by mistake? The link should still point to mysql here.

Reply:
Revert the line back.